modules/hcp-ocp-differences.adoc

```diff
@@ -0,0 +1,168 @@
+|The {hcp} update results in a change to the `.spec.release.image` field in the `HostedCluster` and `NodePools` resources. Any changes to the `ClusterVersion` resource are ignored.
+
+|After you update an {product-title} cluster, both the control plane and worker machines are updated.
+|After you update the hosted cluster, only the control plane is updated. You perform the node pool updates separately.
+
+|===
+
+[id="machine-config-manage_{context}"]
+== Machine configuration and management
+
+[cols="2a,2a",options="header"]
+|===
+
+|{product-title} |{hcp}
+
+|The `MachineSets` resource manages machines in the `openshift-machine-api` namespace.
+|The `NodePool` resource manages machines on the management cluster.
+
+|A set of control plane machines are available.
+|A set of control plane machines do not exist.
+
+|You enable a machine health check by using the `MachineHealthCheck` resource.
+|You enable a machine health check through the `.spec.management.autoRepair` field in the `NodePool` resource.
+
+|You enable autoscaling by using the `ClusterAutoscaler` and `MachineAutoscaler` resources.
+|You enable autoscaling through the `spec.autoScaling` field in the `NodePool` resource.
+
+|Machines and machine sets are exposed in the cluster.
+|Machines, machine sets, and machine deployments from upstream CAPI are used to manage machines but are not exposed to the user.
+
+|All machine sets are upgraded automatically when you update the cluster.
+|You update your node pools independently from the hosted cluster updates.
+
+|Only an in-place upgrade is supported in the cluster.
+|Both replace and in-place upgrades are supported in the hosted cluster.
+
+|The Machine Config Operator manages configuration for machines.
+|The Machine Config Operator does not exist in {hcp}.
+
+|You configure machine Ignition by using the `MachineConfig`, `KubeletConfig`, and `ContainerRuntimeConfig` resources that are selected from a `MachineConfigPool` selector.
+|You configure the `MachineConfig`, `KubeletConfig`, and `ContainerRuntimeConfig` resources through the config map referenced in the `spec.config` field of the `NodePool` resource.
+
+|The Machine Config Daemon (MCD) manages configuration changes and updates on each of the nodes.
+|For an in-place upgrade, the node pool controller creates a run-once pod that updates a machine based on your configuration.
+
+|You can modify the machine configuration resources such as the SR-IOV Operator.
+|You cannot modify the machine configuration resources.
+
+|===
+
+[id="netowrking_{context}"]
+== Networking
+
+[cols="2a,2a",options="header"]
+|===
+
+|{product-title} |{hcp}
+
+|The Kube API server communicates with nodes directly. Because the Kube API server and nodes exist in the same Virtual Private Cloud (VPC).
```

Comment:
```suggestion
|The Kube API server communicates with nodes directly, because the Kube API server and nodes exist in the same Virtual Private Cloud (VPC).
```

Sentence one is dependent on sentence two, so join them with an and clause.